app/src/main/java/ru/krivocraft/kbmp/core/utils/Seconds.java

```diff
@@ -0,0 +1,29 @@
+package ru.krivocraft.kbmp.core.utils;
+
+public class Seconds {
+    private final int seconds;
+
+    public Seconds(int seconds) {
+        //todo: consider throwing IllegalArgumentException for negative values
+        this.seconds = seconds;
+    }
+
+    public String formatted() {
+        int secondsTail = seconds % 60;
+        int minutes = (seconds - secondsTail) / 60;
+
```

Comment:
remove all empty lines, consider extracting sub-methods

Reply:
Refuctoring was intended to be norrow and structural. Nevertheless *done* as  this particular one is small.